src/dmd/ctfeexpr.d

```diff
@@ -235,6 +235,7 @@ extern (C++) final class CTFEExp : Expression
     extern (C++) static __gshared CTFEExp breakexp;
     extern (C++) static __gshared CTFEExp continueexp;
     extern (C++) static __gshared CTFEExp gotoexp;
+    extern (C++) static __gshared CTFEExp errinfoexp;
```

Comment:
I think we should consider if this name is still appropriate.  I suggest something link `showcontext`, `displaycontext`, `printcontext`, or something like that, though `errinfoexp` isn't bad.  Also, since this is a new symbol, it should have a documentation comment.

Reply:
Or even better using the D naming conventions.

Reply:
I would rather have the code I add be consistent with existing code in the same block.